Tasks/Common/defaultClients.ts

```diff
@@ -3,7 +3,18 @@
  * SPDX-License-Identifier: MIT
  */
 
-import { CloudFormation, ECR, IAM, Lambda, S3, SecretsManager, SNS, SQS, SSM } from 'aws-sdk/clients/all'
```

Comment:
Follow up with my above comment, this file name is camel case... This is really inconsistent

Reply:
True, none of the filenames are consistent, we should move towards consistently camelCase though. A rename after these refactors would be a good idea

Reply:
Filed an issue for this: https://github.com/aws/aws-vsts-tools/issues/221